.github/workflows/playwright.yml

```diff
@@ -10,6 +10,22 @@ jobs:
     name: Run Playwright Tests
     timeout-minutes: 60
     runs-on: ubuntu-latest
+    services:
+      minio:
+        image: quay.io/minio/minio
+        ports:
+          - '9000:9000'
+          - '9001:9001'
+        volumes:
+          - 'minio_data:/data'
+        command: |
+          server /data 
+        environment:
+          - MINIO_ROOT_USER=admin
+          - MINIO_ROOT_PASSWORD=admin1234
+          - MINIO_DEFAULT_BUCKETS=meshdb-join-form-log
+          - MINIO_ACCESS_KEY=sampleaccesskey
+          - MINIO_SECRET_KEY=samplesecretkey
```

Comment:
This is a cool application for minio, to act as an S3 mock for testing. I've used getmoto.org for this in python but this is generic across all SDKs, nice